Nessie-iOS-Wrapper/ATM.swift

```diff
@@ -68,9 +68,13 @@ open class ATMRequest {
             if (error != nil) {
                 completion(nil, error)
             } else {
-                let json = JSON(data: data!)
-                let response = AtmResponse(data: json)
-                completion(response, nil)
+                do {
+                    let json = try JSON(data: data!)
```

Comment:
I think we should unwrap safely here to minimize the chances of unwrapping nil. Do you mind giving a default value? I guess nil could work since it should get caught in the catch block.

Reply:
I pushed a new commit. You can't pass nil it as it would be a syntax error, however I passed in Data() in the case data is nil. Let me know if this is ok.